layouts/error.vue

```diff
@@ -1,11 +1,77 @@
 <template>
   <v-app>
-    <h1 v-if="error.statusCode === 404">
-      {{ pageNotFound }}
-    </h1>
-    <h1 v-else>
-      {{ otherError }}
-    </h1>
+    <v-row v-if="error.statusCode === 404">
+      <v-col class="mx-16">
+        <h2>
+          <v-icon color="red" class="mb-1"> 
+            mdi-alert 
+          </v-icon>
+          {{ pageNotFoundEn }}
+        </h2>
+        <p>
+          {{ error404TxtEn }}
+        </p>
+        <li class="mx-10">
+          Return to the
+          <a :href="baseURL">
+            home page
+          </a>
+        </li>
+      </v-col>
+      <v-col class="mx-16">
```

Comment:
Looks like with the `mx-16`, you are trying to do a 50/50 split of the width while keeping it centered for the 2 `v-col`? The x-margin works, but is a bit overkill for it (primary use is for margin spacing along the x-axis).
You can use the `col` property instead: https://vuetifyjs.com/en/components/grids/#breakpoint-sizing

So `col="6"` in the 2 `<v-col>` would make it 50/50 width split by default in all screen sizes. But beware it may look too wide (ie. too much empty whitespace) on a large screen and too squished in a small screen. So you'll need to test by dragging your browser size to see how it looks on different widths and then adjust accordingly.

In a small screen (`xs=` and `sm=`) you can specify something like `xs="12"` and/or `sm="12"` which will fill up the entire width on the smallest screens (which turns the col into a row display).

If it looks too wide on large screens (`lg` and `xl`), you can specify a smaller number less than 6 like `xl="3"` and/or `lg="4"` and also add `justify="center"` to keep it centred (https://vuetifyjs.com/en/components/grids/#justify).